src/osc.h

```diff
@@ -33,9 +34,14 @@ class OSCManager {
       while (reader.isOk() && (msg = reader.popMessage()) != 0) {
         for (const auto &pair : m_callbacks){
           // only call if pattern matches
-          if (msg->match(pair.first).isOk())
+          if (msg->match(pair.first).isOk()){
+            int i; std::string s;
+            msg->arg().popInt32(i).popStr(s);
```

Comment:
we don't wanna pop from the message here. this will leave the message empty, and if the message doesn't have an Int32 or Str in it, this will leave the string empty and the integer empty. 

instead, you should print the message's address pattern (see `const std::string &addressPattern()`) and type tags (see `const std::string &typeTags()`).